src/display/editor/editor.js

```diff
@@ -902,11 +902,9 @@ class AnnotationEditor {
       tooltip.innerText = await AnnotationEditor._l10nPromise.get(
         "editor_alt_text_decorative_tooltip"
       );
-      tooltip.classList.add("decorative");
```

Comment:
With this CSS class being removed, you can probably revert this to what you originally had in a previous patch; i.e. change https://github.com/mozilla/pdf.js/blob/3c0791b5a21ce9936aeb8637b1702c080b9d7e53/src/display/editor/editor.js#L901-L907 into
```javascript
    tooltip.innerText = this.#altTextDecorative
      ? await AnnotationEditor._l10nPromise.get(
          "editor_alt_text_decorative_tooltip"
        )
      : this.#altText;
```